lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -165,6 +179,15 @@ public ElasticIndexingStorage(HttpHost esHost, File tempDir) throws IOException
         this.indexNamePrefix = "";
         this.tempDir = tempDir;
     }
+
```

Comment:
@eapearson Could you please put a javadoc for this new function?

Reply:
Thanks Boris. I added javadocs for the constructors. Can we add docs for the other functions later - you can even assign it to me! - ? I agree they need them -- I had to look far and wide to find examples of Javadoc practices in the codebase! This bug is exposed in the searchapi, so I just want to close the loop on it if everything looks kosher.